### PR TITLE
Schema validate

### PR DESCRIPTION
**Pull request by apfelbox**
\+ bump Fixer dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +45,37 @@ php vendor/bin/php-cs-fixer fix --dry-run --diff --config vendor/becklyn/php-cs/
...
+}
+```
+
+There are two extension points:
+
+*   `getRootDir()` must return the absolute path to the root of your app. By default it assumes that your test is directly
+    in the tests directory top level. Change it, if that isn't the case.
+
+*   `getEntityDirs()` must return the list of absolute paths, where files with mapping info are stored.
+    By default set to `getRootDir()/src/Entity`.
```
Comment (keichinger) [inline README.md:81]: Can we parse here the Symfony config for `doctrine.orm.mappings` as a more flexible default?

File: src/Testing/SchemaValidationTest.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        $config = Setup::createAnnotationMetadataConfiguration(
+            $this->getEntityDirs(),
+            true,
+            null,
+            null,
+            false
+        );
+        $entityManager = EntityManager::create([
+            "driver" => "pdo_sqlite",
+            "path" => __DIR__ . "/db.sqlite",
```
Comment (keichinger) [inline src/Testing/SchemaValidationTest.php]: Is there a reason why we don't want to use SQLite with an in-memory database?

```suggestion
            "memory" => true,
```

File: README.md (hunk 1 above)
Comment (apfelbox) [reply README.md:81]: There is no symfony config in libraries / bundles.

Remember that in projects you normally don't need this command, as there already is the one provided by the doctrine bundle.

File: src/Testing/SchemaValidationTest.php (hunk 2 above)
Comment (apfelbox) [reply src/Testing/SchemaValidationTest.php]: No specific reason, but an in-memory DB is even better.

Thanks 👍

File: README.md (hunk 1 above)
Comment (apfelbox) [reply README.md:81]: I updated the README to clarify that.

**Review by keichinger — APPROVED** [review]

**Pull request merged**